From 4a3e5c6267a49b529ae638f0ac8fc2b03bcc1d36 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Mon, 6 Feb 2023 10:14:28 +0800
Subject: [PATCH 30/45] Revert "drm/etnaviv: add missing quirks for GC300"

This reverts commit 6cd4865bb4bef8fbbb4aac7bc9db6d6a6d256887.
---
 drivers/gpu/drm/etnaviv/etnaviv_gpu.c | 11 ++---------
 1 file changed, 2 insertions(+), 9 deletions(-)

diff --git a/drivers/gpu/drm/etnaviv/etnaviv_gpu.c b/drivers/gpu/drm/etnaviv/etnaviv_gpu.c
index f3281d56b1d82..2520b7dad6ce7 100644
--- a/drivers/gpu/drm/etnaviv/etnaviv_gpu.c
+++ b/drivers/gpu/drm/etnaviv/etnaviv_gpu.c
@@ -408,12 +408,6 @@ static void etnaviv_hw_identify(struct etnaviv_gpu *gpu)
 	if (gpu->identity.model == chipModel_GC700)
 		gpu->identity.features &= ~chipFeatures_FAST_CLEAR;
 
-	/* These models/revisions don't have the 2D pipe bit */
-	if ((gpu->identity.model == chipModel_GC500 &&
-	     gpu->identity.revision <= 2) ||
-	    gpu->identity.model == chipModel_GC300)
-		gpu->identity.features |= chipFeatures_PIPE_2D;
-
 	if ((gpu->identity.model == chipModel_GC500 &&
 	     gpu->identity.revision < 2) ||
 	    (gpu->identity.model == chipModel_GC300 &&
@@ -447,9 +441,8 @@ static void etnaviv_hw_identify(struct etnaviv_gpu *gpu)
 				gpu_read(gpu, VIVS_HI_CHIP_MINOR_FEATURE_5);
 	}
 
-	/* GC600/300 idle register reports zero bits where modules aren't present */
-	if (gpu->identity.model == chipModel_GC600 ||
-	    gpu->identity.model == chipModel_GC300)
+	/* GC600 idle register reports zero bits where modules aren't present */
+	if (gpu->identity.model == chipModel_GC600)
 		gpu->idle_mask = VIVS_HI_IDLE_STATE_TX |
 				 VIVS_HI_IDLE_STATE_RA |
 				 VIVS_HI_IDLE_STATE_SE |
-- 
2.25.1

